### PR TITLE
Loadout Tweaks

### PR DESCRIPTION
**Pull request by whataboutism-alos**
## About The Pull Request

Simple and small pull request which does some tweaks to the loadout menu. Main features are a few needed loadout items which would be ordinarily available roundstart to jobs which are currently disabled on Effigy (Virologist), along with a standardization of grammar and word choice. For example, renaming loadout strings for `Colorable`, `Colourable`, `Greyscale`, `Grayscale`, and `Recolorable` to a uniform `Recolorable`. In some cases, loadout datums have been sorted and repathed to better reflect how the associated item is named, described, or sprited as (such as putting security trousers in misc. undersuit rather than jumpsuit, since it is just pants). _See commits for additional details!_

## Why It's Good For The Game

More options and also uniformity :)

## Changelog

:cl: alos
add: added a few omitted, non-easter egg plushies to loadout selection
add: added virology clothing locked to other medical jobs
add: included additional color variants for certain undersuit and pocket entries
qol: standardized how the loadout selection names recolorable items
fix: fixed mailman uniform datum reusing blue PJs path
spellcheck: fixed loadout item typos
refactor: refactored some loadout item paths to reflect the items better
/:cl:

### CODE REVIEW

File: local/code/datums/loadouts/loadout_datum_pocket.dm [hunk 1]
```diff
@@ -47,44 +47,72 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
...
 	name = "Pack of HP+ Gum"
 	item_path = /obj/item/storage/box/gum/happiness
 
+/datum/loadout_item/pocket_items/gum_pack_nicotine
+	name = "Pack of Nicotine Gum"
+	item_path = /obj/item/storage/box/gum/nicotine
+
+/datum/loadout_item/pocket_items/gum_pack_nicotine
+	name = "Pack of Nicotine Gum"
+	item_path = /obj/item/storage/box/gum/nicotine
```
Comment (unit0016) [inline local/code/datums/loadouts/loadout_datum_pocket.dm:71]: few C+V issues around here

File: local/code/datums/loadouts/loadout_datum_pocket.dm [hunk 2]
```diff
@@ -106,38 +143,43 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
...
 
 /datum/loadout_item/pocket_items/flask
 	name = "Flask"
 	item_path = /obj/item/reagent_containers/cup/glass/flask
 
-/datum/loadout_item/pocket_items/multipen
-	name = "Multicolored Pen"
-	item_path = /obj/item/pen/fourcolor
+/datum/loadout_item/pocket_items/flask	// congrats, you can now start with an alcohol
+	name = "Hooch Bottle"				// bottle to more immersively be the town drunk
```
Comment (unit0016) [inline local/code/datums/loadouts/loadout_datum_pocket.dm:159]: harry dubois 👍

File: local/code/datums/loadouts/loadout_datum_pocket.dm (hunk 1 above)
Comment (whataboutism-alos) [reply local/code/datums/loadouts/loadout_datum_pocket.dm:71]: Ah thank you. I'll get it fixed up for ya!

**Comment by whataboutism-alos** [comment]
There is more I could do here, but I would like to break that work into multiple PRs rather than start pork and barreling this singular one.

**Pull request merged**